components/profile_popover.jsx

```diff
@@ -292,7 +292,7 @@ class ProfilePopover extends React.Component {
         }
 
         const email = this.props.user.email;
-        if (global.window.mm_config.ShowEmailAddress === 'true' || UserStore.isSystemAdminForCurrentUser() || this.props.user === UserStore.getCurrentUser()) {
+        if (email) {
```

Comment:
Yes, the idea behind that is that you would never have the user email returned from the API, if you receive the email with the ShowEmailAddress set to 'false' this would be a backend bug (I'm 3/5 about it).

Reply:
OK, makes sense.